extended_tasks/mt_bench/main.py

```diff
@@ -0,0 +1,138 @@
+# MIT License
+
+# Copyright (c) 2024 The HuggingFace Team
```

Comment:
Is this only our code?

Reply:
yep, I used the original implem but then rewrote pretty much everything for better flexibility and more readable code

Reply:
Sounds good! You can still point to the original repo and say "inspired by" for ex

Reply:
The only file inspired by fastchat is judge.py, I added it there.